uber/migration.py

```diff
@@ -0,0 +1,87 @@
+from collections import OrderedDict
+from os.path import abspath, dirname, join
+
+from alembic import command
+from alembic.config import Config as AlembicConfig
+from alembic.script import ScriptDirectory
+
+from sideboard.internal.imports import plugin_dirs
+
+
+# Path to "alembic.ini", cached here for convenience
+alembic_ini_path = abspath(join(plugin_dirs['uber'], 'alembic.ini'))
```

Comment:
Yeah, I would argue that `plugins` and `plugin_dirs` should be publicly available (or at least a copy of them). As you mentioned, the reason I'm using `plugin_dirs['uber']` is because I necessarily had to import it to generate the `version_locations` below.